neqo-http3/src/connection_client.rs

```diff
@@ -3246,6 +3253,9 @@ mod tests {
             .expect("Set resumption token.");
 
         assert_eq!(client.state(), Http3State::ZeroRtt);
+        let zerortt_event = |e| matches!(e, Http3ClientEvent::StateChange(Http3State::ZeroRtt));
+        assert!(client.events().any(zerortt_event));
+        assert!(client.is_zerortt());
```

Comment:
How does `is_zerortt` differ from `state() == Http3State::ZeroRtt` ?

Reply:
Ok, I can remove is_zerortt. I can implement it in neqo_glue code.